dime-coding-standards/checklists/tex/reviewer_feedback_checklist.tex

```diff
@@ -94,18 +93,17 @@
 	
```

Comment:
Checkbox about tables: suggest wording this as "Tables and charts are exported (preferably as raw text files)"